src/streaming-services.ts

```diff
@@ -0,0 +1,29 @@
+export type StreamingServiceId = 'netflix' | 'nrk' | 'viaplay';
+
+export interface StreamingService {
+	id: StreamingServiceId;
+	name: string;
+	homePage: string;
+	hostPattern: string;
+}
+
+export const streamingServices: Record<StreamingServiceId, StreamingService> = {
+	netflix: {
+		id: 'netflix',
+		name: 'Netflix',
+		homePage: 'https://www.netflix.com/',
+		hostPattern: '*://*.netflix.com/*',
+	},
+	nrk: {
+		id: 'nrk',
+		name: 'NRK',
+		homePage: 'https://tv.nrk.no/',
+		hostPattern: '*://*.tv.nrk.no/*',
+	},
+	viaplay: {
+		id: 'viaplay',
+		name: 'Viaplay',
+		homePage: 'https://viaplay.no/',
```

Comment:
Not relevant for this branch, but have to figure out a way to enable viaplay from other domains, such as .se, .fi, .dk, etc.

Reply:
Since `hostPattern` is responsible for defining the domain, we could just change it to an array of domains instead.